packages/react-dom/src/server/ReactDOMLegacyServerBrowser.js

```diff
@@ -31,10 +31,11 @@ function onError() {
   // Non-fatal errors are ignored.
 }
 
-function renderToStringImpl(
+export function renderToStringImpl(
```

Comment:
This file is a build entry point:

https://github.com/facebook/react/blob/main/scripts/rollup/bundles.js#L273

Exporting something here exposes it to UMD builds as well as the CJS builds.

If you want to share an internal helper you have to move this function to a third module that is imported by both.